Procfile.dev

```diff
@@ -1,3 +1,3 @@
 web: env RUBY_DEBUG_OPEN=true bin/rails server
-js: yarn build --watch
+js: npm run build -- --watch
```

Comment:
`npm run`に`-`で始まるフラグを渡す場合、`--`をつける必要があることに注意。

> Use -- to pass --prefixed flags and options which would otherwise be parsed by npm.

[npm\-run\-script \| npm Docs](https://docs.npmjs.com/cli/v11/commands/npm-run-script)